### ISSUE TITLE
[Accessibility] Testimonial block - Should have no random headings

### ISSUE THREAD

**Issue by andeng1106** [gambitph/Stackable#3255]
**Describe the bug**
- When a Testimonial block is present, there should be no random headings
- Currently, the 
- Should apply to ALL Testimonial layouts

**To Reproduce**
1. Add a new page 
2. Add a Testimonial block (any layout)
3. Select the Name
4. See that it's a Heading block

**Expected behavior**
- Name should not be a Heading block
- The description should be the Heading block (confirm with Jami)

**Screenshots**

https://github.com/user-attachments/assets/13d81597-3897-47e9-809a-e2ef84f23894
